src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/Statistics/TableBuilderController.cs

```diff
@@ -62,31 +63,34 @@ public class TableBuilderController : ControllerBase
             }
 
             return await _tableBuilderService
-                .Query(releaseVersionId, request.AsFullTableQuery(), cancellationToken)
+                .Query(releaseVersionId, request.AsFullTableQuery(), boundaryLevelId, cancellationToken)
                 .HandleFailuresOr(Ok);
         }
 
-        [HttpGet("data/tablebuilder/release/{releaseVersionId:guid}/data-block/{dataBlockParentId:guid}")]
+        [HttpGet("data/tablebuilder/release/{releaseVersionId:guid}/data-block/{dataBlockParentId:guid}/{boundaryLevelId:long?}")]
         public async Task<ActionResult<TableBuilderResultViewModel>> QueryForDataBlock(
             Guid releaseVersionId,
             Guid dataBlockParentId,
+            long? boundaryLevelId,
             CancellationToken cancellationToken = default)
         {
             return await _dataBlockService
                 .GetDataBlockVersionForRelease(releaseVersionId: releaseVersionId, dataBlockParentId: dataBlockParentId)
-                .OnSuccess(dataBlockVersion => GetReleaseDataBlockResults(dataBlockVersion, cancellationToken))
+                .OnSuccess(dataBlockVersion => GetReleaseDataBlockResults(dataBlockVersion, boundaryLevelId, cancellationToken))
```

Comment:
Worth checking formatting on this file for long lines.

Reply:
I run `pnpm format` before raising a PR but that doesn't seem to run on .cs files. Afaik VS doesn't have a line length setting because there's a _visual_ word wrapper which avoids changing the actual code/structure.
![image](https://github.com/user-attachments/assets/471ce447-0b85-4c47-9628-78ce0456a6ab)

Reply:
Ok I remember that we've chatted about this before. Might be worth chatting to @jack-hive to see if he's found a solution? I've added a note to the team meeting board to discuss the differences between IDE's when formatting long lines.